.snippets/code/ethereum-api/ethersjs/deploy.js

```diff
@@ -0,0 +1,45 @@
+// Import ethers and compile
+import { ethers } from 'ethers';
+import contractFile from './compile.js';
+
+// Define network configurations
+const providerRPC = {
+  EvmContainer: {
+    name: 'dancebox-evm-container',
+    rpc: 'https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network', // Insert your RPC URL here
+    chainId: 5678, // 0x162E in hex,
+  },
+};
+
+// Create ethers provider
+const provider = new ethers.JsonRpcProvider(providerRPC.EvmContainer.rpc, {
+  chainId: providerRPC.EvmContainer.chainId,
+  name: providerRPC.EvmContainer.name,
+});
+
+// Define accounts and wallet
+const account_from = {
```

Comment:
please check all `account_from`s and change them to camel case `accountFrom`

Reply:
Ahh good catch! I was actually looking for this but missed some